wagtail_localize/views/edit_translation.py

```diff
@@ -366,6 +367,9 @@ def widget_from_block(block, content_components=None):
         elif isinstance(block, ImageChooserBlock):
             return {"type": "image_chooser"}
 
+        elif isinstance(block, AudioChooserBlock) or isinstance(block, VideoChooserBlock):
+            return {"type": "media_chooser"}
+
```

Comment:
also, need to check whether the direct field (that is the ForeignKey) is to a media class and set the widget type for it as well in `widget_from_field()`, a la https://github.com/wagtail/wagtail-localize/blob/b619a32cc6410fec28a27c0ce1e305605dcb44c0/wagtail_localize/views/edit_translation.py#L321-L322